static/app/views/alerts/rules/metric/details/metricChart.tsx

```diff
@@ -87,7 +88,7 @@ type Props = WithRouterProps & {
   rule: MetricRule;
   timePeriod: TimePeriodType;
   incidents?: Incident[];
-  isOnDemandMetricAlert?: boolean;
+  onDemandMetricAlert?: boolean;
```

Comment:
why remove the `is`?
It would be nice if the name can already point out that it is a boolean.